components/server/ee/src/user/stripe-service.ts

```diff
@@ -61,7 +61,9 @@ export class StripeService {
             email: User.getPrimaryEmail(user),
             name: User.getName(user),
             metadata: {
+                // userId is deprecated, use attributionId where possible
                 userId: user.id,
```

Comment:
Nit: I think the only place where this `userId` metadata is used is a few lines above in `findCustomerByUserId`.

If you also update `findCustomerByUserId` to use the `attributionId`, then you can completely remove the now useless `userId` metadata. 👍

```suggestion
```